### PR TITLE
Social Zombie

### PR DESCRIPTION
**Pull request by NanopassKai**

### CODE REVIEW

File: contracts/MetaverseBaseNFT.sol [hunk 1]
```diff
@@ -339,23 +340,32 @@ contract MetaverseBaseNFT is
...
 
         address payable dev = DEVELOPER_ADDRESS();
+        address payable oxy = OXY_ADDRESS();
 
-        Address.sendValue(payable(msg.sender), amount);
-        Address.sendValue(dev, balance - amount);
+        Address.sendValue(dev, devAmount);
+        Address.sendValue(oxy, oxyAmount);
+        
+        uint256 finalBalance = address(this).balance;
```
Comment (caffeinum) [inline contracts/MetaverseBaseNFT.sol:352]: I would do
```suggestion
        uint256 finalBalance = balance - devAmount - oxyAmount;
```
to save on gas a bit

**Comment by theshadowagent** [comment]
Can you create a different file that inherits from `MetaverseBaseNFT.sol`?

**Comment by NanopassKai** [comment]
don't use this, change of agreement

**Comment by NanopassKai** [comment]
can proceed with original contract

**Pull request closed without merging**